config/prow/perfdash/perfdash-ingress.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     kubernetes.io/ingress.class: public-iks-k8s-nginx
 spec:
   rules:
-    - host: perf-dash.ppc64le-cloud.org
+    - host: perfdash.ppc64le-cloud.cis.ibm.net
```

Comment:
any reason for renaming this to `perfdash`?

Reply:
Right, it has to be perf-dash itself! https://github.com/ppc64le-cloud/test-infra/issues/318

I thought the other way around. Will change back.